### PR TITLE
[Kotlin Migration] ReviewerNoParamTest

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt [hunk 1]
```diff
@@ -13,386 +13,357 @@
...
-        setGestureSetting(false);
+    private fun disableGestureSetting() {
+        setGestureSetting(false)
     }
 
-    private void setGestureSetting(boolean value) {
-        Editor settings = AnkiDroidApp.getSharedPrefs(getTargetContext()).edit();
-        settings.putBoolean(GestureProcessor.PREF_KEY, value);
-        settings.apply();
+    @KotlinCleanup(".edit {}")
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt:267]: I thought I'd caught something, awesome, thanks so much!!!

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt [hunk 2]
```diff
@@ -13,386 +13,357 @@
...
-    private String getKey(Gesture gesture) {
-        switch (gesture) {
-            case SWIPE_UP: return "gestureSwipeUp";
-            case SWIPE_DOWN: return "gestureSwipeDown";
-            case SWIPE_LEFT: return "gestureSwipeLeft";
-            case SWIPE_RIGHT: return "gestureSwipeRight";
-            default: throw new NotImplementedException(gesture.toString());
+    private fun getKey(gesture: Gesture): String {
+        return when (gesture) {
+            Gesture.SWIPE_UP -> "gestureSwipeUp"
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt:295]: import `Gesture`

**Review by david-allison — CHANGES_REQUESTED** [review]
Awesome! One nitpick

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt (hunk 2 above)
Comment (BrayanDSO) [reply AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt:295]: Gesture is imported. Should I directly import each gesture?

Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/ReviewerNoParamTest.kt:295]: > Gesture is imported. Should I directly import each gesture?

As it was on the 'before' side, so yes, or import `*` if feasible.

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
:rocket:

**Pull request merged**